bcda/auth/alpha_test.go

```diff
@@ -53,18 +53,14 @@ func (s *AlphaAuthPluginTestSuite) TestRegisterClient() {
 	c, err := s.p.RegisterClient(auth.KnownFixtureACO)
 	assert.Nil(s.T(), err)
 	assert.NotNil(s.T(), c)
+	assert.NotNil(s.T(), c.ClientSecret)
 	assert.Equal(s.T(), auth.KnownFixtureACO, c.ClientID)
 
 	c, err = s.p.RegisterClient("")
 	assert.NotNil(s.T(), err)
 	assert.Empty(s.T(), c.ClientID)
 	assert.Contains(s.T(), err.Error(), "provide a non-empty string")
 
```

Comment:
We should add a test for making sure we can successfully compare the hashed secret in the db to the one received in the result.